src/main/resources/assets/carpet/scripts/draw.sc

```diff
@@ -0,0 +1,196 @@
+import('math','_euclidean_sq','_vec_length');
+
+__config() -> {
+    'commands'->{
+        'sphere <center> <radius> <block>'->['draw_sphere',null, false],
+        'sphere <center> <radius> <block> replace <replacement>'->['draw_sphere',false],
+        'ball <center> <radius> <block>'->['draw_sphere', null, true],
+        'ball <center> <radius> <block> replace <replacement>'->['draw_sphere', true],
+        'diamond <center> <radius> <block>'->['draw_diamond',null],
+        'diamond <center> <radius> <block> replace <replacement>'->'draw_diamond',
+        'cone <center> <radius> <height> <pointing> <orientation> <block> <hollow>'->['draw_pyramid',null, false],
+        'cone <center> <radius> <height> <pointing> <orientation> <block> <hollow> replace <replacement>'->['draw_pyramid', false],
+        'pyramid <center> <radius> <height> <pointing> <orientation> <block> <hollow>'->['draw_pyramid',null, true],
+        'pyramid <center> <radius> <height> <pointing> <orientation> <block> <hollow> replace <replacement>'->['draw_pyramid', true],
+        'cylinder <center> <radius> <height> <orientation> <block> <hollow>'->['draw_prism', null, false],
+        'cylinder <center> <radius> <height> <orientation> <block> <hollow> replace <replacement>'->['draw_prism', false],
+        'cuboid <center> <radius> <height> <orientation> <block> <hollow>'->['draw_prism', null, true],
+        'cuboid <center> <radius> <height> <orientation> <block> <hollow> replace <replacement>'->['draw_prism', true]
+    },
+    'arguments'->{
+        'center'->{'type'->'pos', 'loaded'->'true'},
+        'radius'->{'type'->'int', 'suggest'->[]},//to avoid default suggestions
+        'replacement'->{'type'->'block'},
+        'height'->{'type'->'int', 'suggest'->[]},
+        'orientation'->{'type'->'string', 'suggest'->['x','y','z']},
+        'pointing'->{'type'->'string','suggest'->['up','down']},
+        'hollow'->{'type'->'bool'}
```

Comment:
since its a last arument - just leave it as a literal, right?

Reply:
last arg is replacement...

Reply:
Anyway, hollow could be a string to make it simpler to understand when brigadier wants you to not see things (and doesn't tell you the arg name, just true/false) (`hollow`|`(whatever is the opposite to that and isn't similar to other args)`)